src/test/java/tests/HelpTest.java

```diff
@@ -0,0 +1,58 @@
+package tests;
+
+import Page_Objects.TestBase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+
+public class HelpTest {
+
+    private WebDriver driver;
+    private TestBase testBase;
+    private Actions actions;
+
+
+    @BeforeEach
+    public void setUp() {
+        driver = new ChromeDriver();
+        testBase = new TestBase(driver);
+        actions = new Actions(driver);
+        testBase.setUp();
+    }
+
+    @Test
+    public void hidingAndOpeningHelpContainer() throws InterruptedException {
+        testBase.getLandingPage().enterTheGame();
+        Thread.sleep(1000);

```

Comment:
sleeps are usually a bad practice and should be avoided. If it's necessary here, leave a comment why